DragonToast/Display/ToastFrame.lua

```
@@ -20,43 +20,44 @@ local GetTime = GetTime
+local Utils = ns.ListenerUtils
+
+local DEFAULT_TEXT_PADDING_V = 6
+local DEFAULT_TEXT_PADDING_H = 8
+local DEFAULT_BORDER_SIZE = 1
+local DEFAULT_GLOW_WIDTH = 4
+local DEFAULT_ICON_SIZE = 36
+local DEFAULT_SECONDARY_FONT_SIZE = 10
+local DEFAULT_FRAME_WIDTH = 350
+local DEFAULT_FRAME_HEIGHT = 48
+local BASE_FRAME_LEVEL = 100
+local HOVERED_FRAME_STRATA = "HIGH"
```

Luacheck (1.2.0)

[warning] unused variable 'HOVERED_FRAME_STRATA'

(W211)